composer.json

```diff
@@ -127,9 +127,9 @@
         "league/flysystem-aws-s3-v3": "Required to use the Flysystem S3 driver (^1.0).",
         "league/flysystem-cached-adapter": "Required to use the Flysystem cache (^1.0).",
         "league/flysystem-sftp": "Required to use the Flysystem SFTP driver (^1.0).",
-        "psr/http-message": "Required to allow Storage::put to accept a StreamInterface (^1.0)",
         "moontoast/math": "Required to use ordered UUIDs (^1.1).",
         "pda/pheanstalk": "Required to use the beanstalk queue driver (^4.0).",
+        "psr/http-message": "Required to allow Storage::put to accept a StreamInterface (^1.0)",
```

Comment:
Is this sentence required to end with dot as any others? :)

Reply:
Good spot. Fixed.